resources/definitions/stream30pro_mk2.def.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": 2,
+    "name": "VOLUMIC STREAM30PRO MK2",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "VOLUMIC",
+        "manufacturer": "VOLUMIC",
+        "file_formats": "text/x-gcode",
+        "icon": "volumic-icon",
+        "platform": "STREAM30PRO_platform.STL",
+        "has_materials": true,
+        "has_machine_quality": true,
+        "machine_extruder_trains":{"0": "stream30_extruder"}
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "VOLUMIC STREAM30PRO MK2" },
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 300 },
+        "machine_height": { "default_value": 300 },
```

Comment:
This printer has a height of 310mm, doesn't it?
https://imprimante-3d-volumic.com/en/3d-printer-range-detail-stream-30-pro-mk2/

Reply:
this is the correct value (300), the mistake is in our website, we will correct this asap.